### PR TITLE
EB Definition - h-periodic

### PR DESCRIPTION
**Pull request by indigobravo**
In an attempted to [Fix h-periodic failing deploys](https://github.com/hypothesis/deployment/issues/151). This branch upgrades the version of Amazon Linux we are using for h-periodic in both QA and PROD.

An update has also been made to the bless ebextension. The new version of Linux does not come with pip2, and that is a requirement for the bless configuration.

Once these updates have been reviewed the changes will be pushed to QA before being rolled out to PROD.

### CODE REVIEW

File: h-periodic/ebextensions/prod/90_bless.config [hunk 1]
```diff
@@ -58,6 +58,8 @@ files:
...
 
 commands:
+  96_install_pip:
```
Comment (indigobravo) [inline h-periodic/ebextensions/prod/90_bless.config:61]: New command introduced to install pip

File: h-periodic/ebextensions/qa/90_bless.config [hunk 2]
```diff
@@ -58,6 +58,8 @@ files:
...
 
 commands:
+  96_install_pip:
```
Comment (indigobravo) [inline h-periodic/ebextensions/qa/90_bless.config:61]: New command introduced to install pip

File: h-periodic/env-prod.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 AWSConfigurationTemplateVersion: 1.1.0.0
 Platform:
-  PlatformArn: arn:aws:elasticbeanstalk:us-west-1::platform/Docker running on 64bit Amazon Linux/2.14.0
+  PlatformArn: arn:aws:elasticbeanstalk:us-west-1::platform/Docker running on 64bit Amazon Linux 2/3.2.0
```
Comment (indigobravo) [inline h-periodic/env-prod.yml:3]: Definition of the new Linux version.

File: h-periodic/env-qa.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 AWSConfigurationTemplateVersion: 1.1.0.0
 Platform:
-  PlatformArn: arn:aws:elasticbeanstalk:us-west-1::platform/Docker running on 64bit Amazon Linux/2.7.0
+  PlatformArn: arn:aws:elasticbeanstalk:us-west-1::platform/Docker running on 64bit Amazon Linux 2/3.2.0
```
Comment (indigobravo) [inline h-periodic/env-qa.yml:3]: Definition of the new Linux version.

**Review by seanh — APPROVED** [review]
Let's give it a try

**Pull request merged**